api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2.java

```diff
@@ -36,7 +37,8 @@
                 names = @Crd.Spec.Names(
                         kind = KafkaMirrorMaker2.RESOURCE_KIND,
                         plural = KafkaMirrorMaker2.RESOURCE_PLURAL,
-                        shortNames = {KafkaMirrorMaker2.SHORT_NAME}
+                        shortNames = {KafkaMirrorMaker2.SHORT_NAME},
+                        categories = {STRIMZI_CATEGORY}
```

Comment:
You have to build `api` module and do `make crd_install` to propagate this change into CRD files.

Reply:
Fixed, thank you!